app/notifications/process_notifications.py

```diff
@@ -335,20 +334,11 @@ def send_to_queue_for_recipient_info_based_on_recipient_identifier(
     This is the execution path for sending notifications with recipient identifiers.
     """
 
-    if id_type == IdentifierType.VA_PROFILE_ID.value:
-        tasks = [
-            send_va_onsite_notification_task.s(id_value, str(notification.template.id), onsite_enabled).set(
-                queue=QueueNames.NOTIFY
-            ),
-        ]
-
-    else:
-        tasks = [
+    tasks = []
+    if id_type != IdentifierType.VA_PROFILE_ID.value:
+        tasks.append(
```

Comment:
This is the only real change in this PR. Removed ability to enqueue the onsite notification task.